### ISSUE TITLE
Warp quiet param seems does not exists

### ISSUE THREAD

**Issue by latot** [USDAForestService/gdalraster#251]
Hi, this is weird, seems the quiet param of warp does not exists:

```R
ds <- system.file("extdata/storml_tcc.tif", package="gdalraster")
gdalraster::warp(
  ds,
  "/vsimem/test.tif",
  "",
  quiet = TRUE
)
Error in gdalraster::warp(ds, "/vsimem/test.tif", "", quiet = TRUE) : 
  unused argument (quiet = TRUE)
```

Thx!

**Comment by ctoney** [comment]
Hi, are you using the released version from CRAN (1.9.0)? The website documentation is tracking the development version of gdalraster, with a changelog maintained at:
https://usdaforestservice.github.io/gdalraster/news/index.html

The quiet param was added recently (on 2024-02-27 at dev version 1.9.0.9040). The dev version is generally stable and could be installed from source from GitHub, or from the binaries built on r-universe:
```r
install.packages("gdalraster", repos = c("https://usdaforestservice.r-universe.dev", "https://cran.r-project.org"))
```
Otherwise, another CRAN update is planned for later this month. Sorry for the confusion. We should probably better indicate the documentation status on the website.

**Comment by latot** [comment]
you are right! sorry I should have checked that :)
thx
